opencontext/server/routes/debug.py

```
@@ -484,99 +484,99 @@ async def update_debug_prompts(
 @router.post("/api/debug/generate/{category}/custom")
 async def generate_with_custom_prompts(
     category: str,
-    lookback_minutes: int = Query(15, description="回顾分钟数"),
-    start_time: int = Query(None, description="开始时间戳"),
-    end_time: int = Query(None, description="结束时间戳"),
+    lookback_minutes: int = Query(15, description="Lookback minutes"),
+    start_time: int = Query(None, description="Start timestamp"),
+    end_time: int = Query(None, description="End timestamp"),
     opencontext: OpenContext = Depends(get_context_lab),
```

Ruff (0.13.3)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)